src/app/components/footer/footer.html

```diff
@@ -1,49 +1,70 @@
3.62 4.93 4.93 0 0 1-1.52-6.57 14 14 0 0 0 10.15 5.14 5 5 0 0 1-.13-1.12 4.93 4.93 0 0 1 8.52-3.37 9.83 9.83 0 0 0 3.13-1.19 4.94 4.94 0 0 1-2.16 2.72 9.83 9.83 0 0 0 2.78-.78 10 10 0 0 1-2.46 2.55z"/></symbol><symbol viewBox="0 0 48 48" id="sm-yt-def" xmlns="http://www.w3.org/2000/svg"><path d="M32.58 16.83c-1.72-.45-8.59-.45-8.59-.45s-6.75 0-8.46.42a3 3 0 0 0-2.11 2A26.77 26.77 0 0 0 13 24a26.77 26.77 0 0 0 .42 5.24 3 3 0 0 0 2.11 2c1.72.46 8.46.42 8.46.42s6.75 0 8.46-.42a3 3 0 0 0 2.08-2A27.91 27.91 0 0 0 35 24a28.42 28.42 0 0 0-.47-5.24 2.74 2.74 0 0 0-1.95-1.93zM21.26 27.38v-6.76L26.74 24z"/><path d="M24 2a22 22 0 1 0 22 22A22 22 0 0 0 24 2zm0 41a19 19 0 1 1 19-19 19 19 0 0 1-19 19z"/></symbol><symbol viewBox="0 0 48 48" id="sm-yt-full" xmlns="http://www.w3.org/2000/svg"><path d="M24 2a22 22 0 1 0 22 22A22 22 0 0 0 24 2zm10.54 27.24a3 3 0 0 1-2.08 2c-1.72.45-8.46.42-8.46.42s-6.75 0-8.46-.42a3 3 0 0 1-2.11-2A26.77 26.77 0 0 1 13 24a26.77 26.77 0 0 1 .42-5.24 3 3 0 0 1 2.11-2c1.72-.45 8.46-.42 8.46-.42s6.87 0 8.59.45a2.74 2.74 0 0 1 1.94 1.93A28.42 28.42 0 0 1 35 24a27.91 27.91 0 0 1-.46 5.24z"/><path d="M21.25 27.38l5.5-3.38-5.5-3.38v6.76z"/></symbol><symbol viewBox="0 0 48 48" id="trending" xmlns="http://www.w3.org/2000/svg"><path d="M47.88 16.18l-4-14a3 3 0 0 0-3.7-2.07l-14 4a3 3 0 1 0 1.65 5.77l5.4-1.54-10.14 18.09-4.26-4.26a4 4 0 0 0-6.21.69l-12 19a4 4 0 1 0 6.76 4.27l9.33-14.77 4.46 4.46a4 4 0 0 0 6.32-.82l12.92-23.13 1.7 6a3 3 0 0 0 5.77-1.65z"/></symbol><symbol viewBox="0 0 48 48" id="upload" xmlns="http://www.w3.org/2000/svg"><path d="M40.12 14.88l-14-14a3 3 0 0 0-4.24 0l-14 14a3 3 0 0 0 4.24 4.24L21 10.24V36a3 3 0 0 0 6 0V10.24l8.88 8.88a3 3 0 0 0 4.24-4.24z"/><path d="M45.8 34H30v2a6 6 0 0 1-12 0v-2H2.2A2.37 2.37 0 0 0 0 36.51v9A2.37 2.37 0 0 0 2.2 48h43.6a2.37 2.37 0 0 0 2.2-2.51v-9A2.37 2.37 0 0 0 45.8 34zM36 43a2 2 0 1 1 2-2 2 2 0 0 1-2 2zm7 0a2 2 0 1 1 2-2 2 2 0 0 1-2 2z"/></symbol><symbol viewBox="0 0 48 48" id="user" xmlns="http://www.w3.org/2000/svg"><path d="M23 24h2a16 16 0 0 1 16 16v4H7v-4a16 16 0 0 1 16-16z"/><rect x="15" y="4" width="18" height="18" rx="9" ry="9"/></symbol><symbol viewBox="0 0 48 48" id="user--white" xmlns="http://www.w3.org/2000/svg"><path fill="#fff" d="M23 24h2a16 16 0 0 1 16 16v4H7v-4a16 16 0 0 1 16-16z"/><rect fill="#fff" x="15" y="4" width="18" height="18" rx="9" ry="9"/></symbol><symbol viewBox="0 0 48 48" id="users" xmlns="http://www.w3.org/2000/svg"><rect x="9" y="7" width="16" height="16" rx="8" ry="8"/><rect x="27" y="11" width="12" height="12" rx="6" ry="6"/><path d="M34 25h-2a11 11 0 0 0-3.8.69A16 16 0 0 1 34 38v1h11v-3a11 11 0 0 0-11-11zM16 25h2a13 13 0 0 1 13 13v3H3v-3a13 13 0 0 1 13-13z"/></symbol><symbol viewBox="0 0 48 48" id="warning" xmlns="http://www.w3.org/2000/svg"><path d="M37 2H11a8 8 0 0 0-8 8v19a8 8 0 0 0 8 8h14l8.8 8.81a.7.7 0 0 0 1.2-.51V37h2a8 8 0 0 0 8-8V10a8 8 0 0 0-8-8zm-15 8a2 2 0 1 1 4 0v11a2 2 0 0 1-4 0zm2 21a2.5 2.5 0 1 1 2.5-2.5A2.5 2.5 0 0 1 24 31z"/></symbol></svg>
```

Comment:
@left23 I noticed that adding the SVG (regardless of where or how) caused a collision in this codebase. The SVG contains a `<symbol id="search">` and we also have an input with `id="search"` in the UI.

I don't want to change the CD sprite right now, but wanted to highlight this as something we might want to change in the future by namespacing our HTML IDs, particularly when they are so generic and common like "search" or "back" etc.

More info: https://humanitarian.atlassian.net/browse/HID-1832

Reply:
Well spotted. The IDs are coming from the SVG file names but there are config options for prefixing we should avail of https://github.com/jkphl/svg-sprite#common-mode-properties

Reply:
ooh interesting that we could potentially prefix per-project. For now I've simply done a compile within the canonical `ocha_basic` project and copied the assets over.

However it's not that much extra work to compile with a prefix and re-copy assets (along with instructions in the README of course!). Thanks for pointing me to the spriting docs!